### PR TITLE
ecl: add ecl-devel subport

### PR DESCRIPTION
**Pull request by catap**
###### Type(s)

- [x] enhancement

###### Tested on

macOS 12.6.8 21G725 x86_64
Xcode 14.2 14C18

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: _resources/port1.0/group/common_lisp-1.0.tcl
Comment (catap) [inline _resources/port1.0/group/common_lisp-1.0.tcl:83]: @mascguy I not sure how to make it cleaner. Do you have any suggestion?

Comment (mascguy) [reply _resources/port1.0/group/common_lisp-1.0.tcl:83]: > @mascguy I not sure how to make it cleaner. Do you have any suggestion?

Are you referring to the combinations of `ecl` and `ecl-devel`, which you're deleting from `depends_build`?

Comment (catap) [reply _resources/port1.0/group/common_lisp-1.0.tcl:83]: @mascguy yes, I'm

Comment (mascguy) [reply _resources/port1.0/group/common_lisp-1.0.tcl:83]: I think it's fine as-is.

**Review by mascguy — APPROVED** [review]

**Comment by mascguy** [comment]
LGTM, but I'll wait for your final go-ahead before merging.

**Comment by catap** [comment]
@mascguy let's merge it!

**Comment by catap** [comment]
Thanks!

**Pull request merged**
